scripts/lib-franklin.js

```diff
@@ -366,13 +366,43 @@ export function decorateSupScript(string, result = []) {
   return result;
 }
 
+export function getInfo() {
+  const [, country, language] = window.location.pathname.split('/');
+  return {
+    country,
+    language,
+    productDB: new URL('/products.json', window.location).pathname,
+    productSupport: new URL(`/${country}/${language}/product-support`, window.location).pathname,
+    queryIndex: new URL(`/${country}/${language}/query-index.json`, window.location).pathname,
+  };
+}
+
+export function adjustAssetURL(asset) {
+  if (asset?.URL) {
+    const url = new URL(asset.URL, window.location);
+    if (url.hostname.endsWith('-mammotome--hlxsites.hlx.page')
```

Comment:
you could shorten this slightly by having an array of host names. the array values could even be a constant somewhere. using an array would make it easier to add/remove domains:
```
    const isLocalOrHlx = ['localhost', '-mammotome--hlxsites.hlx.page', '-mammotome--hlxsites.hlx.live']
      .some(domain => url.hostname.endsWith(domain));
      
    if (isLocalOrHlx) {
      asset.URL = url.pathname;
    }
```